plugins/postcss-nesting/README.md

```diff
@@ -80,43 +82,28 @@ instructions for:
 - [Gulp](INSTALL.md#gulp)
 - [Grunt](INSTALL.md#grunt)
 
-## ⚠️ `@nest` has been removed from the specification.
-
-Previous iterations of the [CSS Nesting specification] required using `@nest` for certain selectors.
-
-`@nest` was removed from the specification completely.
-Future versions of this plugin will error if you use `@nest`.
-
-We advice everyone to migrate their codebase **now** to nested CSS without `@nest`.  
-We published a [Stylelint Plugin](https://github.com/csstools/postcss-plugins/tree/main/plugins-stylelint/no-at-nest-rule#csstoolsstylelint-no-at-nest-rule) to help you migrate.
-
-example warning:
-> `@nest` was removed from the CSS Nesting specification and will be removed from PostCSS Nesting in the next major version.
-> Change `@nest foo & {}` to `foo & {}` to migrate to the latest standard.
-
-You can silence this warning with a new `silenceAtNestWarning` plugin option.
-
-```js
-postcssNesting({
-	silenceAtNestWarning: true
-})
-```
-
 ## Options
 
 ### edition
 
-The default behavior is to transpile CSS following an older version of the CSS nesting specification.
+The default behavior is to transpile CSS following the version of the CSS nesting specification as it existed during the last major version of this plugin.
 
-If you want to already use the latest version you can set the `edition` option to `2024-02`.
+If you want to use an older or even more recent version you can set the `edition` option.
```

Comment:
We don't have the option right now to do "even more recent"?

Reply:
Correct! I was again trying to future proof these statements, but I agree that this is mostly making it more confusing :)